get-started/part4.md

```diff
@@ -31,6 +31,14 @@ username/repo:tag`, then visit `http://localhost/`.
 
 - Have a copy of your `docker-compose.yml` from [Part 3](part3.md) handy.
 
+> This part of the tutorial requires command line `scp`.
```

Comment:
This part of the tutorial's use of `scp` is misguided and counterproductive, Windows or no. I recommend it be refactored. You should just use `docker-machine env` (if I remember correctly) and then `docker stack deploy` directly from the host, instead of shoving the docker-compose file in with `scp`

Reply:
Okay I'll change it to `docker-machine env`, just cc'ing @johndmulhausen as an fyi

Reply:
+1 since it decreases the dependency on distribution-specific tools and shows that Docker has a client-server architecture and you can use the client to manage any server/swarm